### PR TITLE
ensure --show_hidden is used in the correct location for 'avail' with Lmod

### PR DESCRIPTION
**Pull request by boegel**
fix for bug reported by @geimer

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/tools/modules.py [hunk 1]
```diff
@@ -579,19 +579,47 @@ def check_module_output(self, cmd, stdout, stderr):
...
+
+        :param args: list of arguments for module command
+        :return: list of strings representing the full module command to run
+        """
+        if opts is None:
+            opts = []
+
+        cmdlist = [self.cmd, 'python']
+
+        if args[0] in ('available', 'avail', 'list',):
```
Comment (wpoely86) [inline easybuild/tools/modules.py]: `av`?

Comment (boegel) [inline easybuild/tools/modules.py]: EasyBuild never uses `av` as subcommand... (see implementation of `available` method)

**Comment by boegel** [comment]
@wpoely86 any other remarks, or is this considered good to go?

**Comment by wpoely86** [comment]
good to go for me

**Comment by boegel** [comment]
Thanks @wpoely86!

**Pull request merged**